specs/experimental/plasma.md

```diff
@@ -180,6 +190,9 @@ In addition, an expired challenge will reorg out `[r_start, r_end]` L2 blocks so
 block derived from the expired challenge's input and `r_end` the last L2 block derived before the pipeline
 was reset.
 
+Derivation MUST skip input data such as `input_data_size > max_l1_calldata_size` to ensure data can be submitted
```

Comment:
This should be included in a constants table that includes the value

Reply:
The max size has to be a consensus constant that is enforced in the derivation pipeline to guarantee that everything can be challenged. Otherwise you need to use a streaming keccak implementation like in the dispute game which is complex